src/java/fr/paris/lutece/plugins/workflow/modules/appointmentants/business/TaskAntsApplicationNumberDAO.java

```diff
@@ -0,0 +1,45 @@
+package fr.paris.lutece.plugins.workflow.modules.appointmentants.business;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import fr.paris.lutece.plugins.workflow.modules.appointmentants.service.WorkflowAppointmentAntsPlugin;
+import fr.paris.lutece.util.sql.DAOUtil;
+
+public class TaskAntsApplicationNumberDAO {
+	
+	public static final String BEAN_NAME = "workflow-appointmentants.taskAntsApplicationNumberDAO";
+	
+	/**
+	 * SQL Queries
+	 */
+	private static final String SQL_QUERY_GET_ANTS_APPLICATION_VALUES = "SELECT response_value FROM appointment_appointment aa "
```

Comment:
It is not advisable to directly query the tables of one plugin from another. To retrieve data from the "appointment" plugin, it is better to go through its service layer.